### ISSUE TITLE
Sync stalls on phase0 blocks with LH v2.2.0

### ISSUE THREAD

**Issue by jdevcs** [sigp/lighthouse#3147]
## Description

Beacon node is unable to sync and `start_slot` is always 0 

## Version

using docker image: latest or v2.2.0

## Present Behavior

`curl --location --request GET 'http://localhost:5052/lighthouse/syncing'`

its always responding

```
{
    "data": "Stalled"
}

```
and is at start_slot 0

```
{
    "data": {
        "SyncingFinalized": {
            "start_slot": "0",
            "target_slot": "3543488"
        }
    }
}
```

## Expected Behaviour

start_slot should increase 

## Steps to resolve

For simple reproducing: 
`docker run -p 9000:9000 -p 127.0.0.1:5052:5052 -v ${HOME}/.lighthouse:/root/.lighthouse1 sigp/lighthouse lighthouse --network mainnet beacon --http --http-address 0.0.0.0`

**Comment by pawanjay176** [comment]
Thanks for the issue. We are testing a fix with this PR https://github.com/sigp/lighthouse/pull/3146

If you are syncing fresh, then I recommend downgrading to `v2.1.5`

**Comment by michaelsproul** [comment]
Closing as this is fixed in v2.2.1, but I'll leave this pinned a bit longer
